assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/hooks/useOnSubmitPrices.ts

```diff
@@ -59,7 +59,9 @@ const useOnSubmitPrices = (): VoidFunction => {
 		// Delete all unlucky ones
 		await Promise.all(
 			deletedPriceIds.map((id) => {
-				return new Promise((onCompleted, onError) => deletePrice({ id }, { onCompleted, onError }));
+				return new Promise((onCompleted, onError) =>
+					deletePrice({ id, deletePermanently: true }, { onCompleted, onError })
```

Comment:
I have set the prices to be deleted permanently by default

Reply:
totally makes sense considering we have no UI for managing trashed prices 👍